desktop_native/src/biometric/win.rs

```diff
@@ -0,0 +1,66 @@
+use windows::core::*;
+use windows::Foundation::IAsyncOperation;
+use windows::Security::Credentials::UI::*;
+use windows::Win32::Foundation::HWND;
+use windows::Win32::System::WinRT::IUserConsentVerifierInterop;
+
+pub async fn available() -> bool {
+    let event = UserConsentVerifier::CheckAvailabilityAsync();
+    let result = match event {
+        Err(_) => return false,
+        Ok(t) => t.await,
+    };
+
+    match result {
+        Err(_) => false,
+        Ok(t) => match t {
+            UserConsentVerifierAvailability::Available => true,
+            UserConsentVerifierAvailability::DeviceBusy => true,
+            _ => false,
+        },
+    }
+}
+
+pub async fn verify(
+    message: &str,
+    window_handle: isize,
+) -> std::result::Result<bool, Box<dyn std::error::Error + Send + Sync>> {
```

Comment:
I think `anyhow::Result` is much friendlier than this https://docs.rs/crate/anyhow/1.0.26

Reply:
Yes most likely. I wanted to avoid adding further dependencies while hacking it togheter. Certanly something we should look into cleaning up.